src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -26490,73 +26776,1152 @@ void Validate(ModuleSymbol m)
+                // class Test1 : I2, I3
+                Diagnostic(ErrorCode.ERR_MostSpecificImplementationIsNotFound, "I3").WithArguments("I1<string?>.M1()", "I2.I1<string>.M1()", "I3.I1<string?>.M1()").WithLocation(2, 19)
+                );
+
+            var test1 = compilation2.GetTypeByMetadataName("Test1");
+
+            Assert.Equal(new[] { "I2", "I1<System.String>", "I3", "I1<System.String?>" },
+                         test1.AllInterfacesNoUseSiteDiagnostics.Select(i => i.ToTestDisplayString()));
+
+            Assert.Null(test1.FindImplementationForInterfaceMember(test1.AllInterfacesNoUseSiteDiagnostics[1].GetMember("M1")));
+            Assert.Null(test1.FindImplementationForInterfaceMember(test1.AllInterfacesNoUseSiteDiagnostics[3].GetMember("M1")));
+
+            var compilation3 = CreateCompilation(source2, new[] { compilation1.EmitToImageReference() }, options: TestOptions.DebugDll,
+                                                    parseOptions: TestOptions.Regular,
+                                                    targetFramework: TargetFramework.NetStandardLatest);
+
+            compilation3.VerifyDiagnostics(
+                // (2,7): warning CS8645: 'I1<string?>' is already listed in the interface list on type 'Test1' with different nullability of reference types.
+                // class Test1 : I2, I3
+                Diagnostic(ErrorCode.WRN_DuplicateInterfaceWithNullabilityMismatchInBaseList, "Test1").WithArguments("I1<string?>", "Test1").WithLocation(2, 7),
+                // (2,15): error CS8705: Interface member 'I1<string>.M1()' does not have a most specific implementation. Neither 'I2.I1<string>.M1()', nor 'I3.I1<string>.M1()' are most specific.
+                // class Test1 : I2, I3
+                Diagnostic(ErrorCode.ERR_MostSpecificImplementationIsNotFound, "I2").WithArguments("I1<string>.M1()", "I2.I1<string>.M1()", "I3.I1<string>.M1()").WithLocation(2, 15),
+                // (2,19): error CS8705: Interface member 'I1<string?>.M1()' does not have a most specific implementation. Neither 'I2.I1<string>.M1()', nor 'I3.I1<string>.M1()' are most specific.
+                // class Test1 : I2, I3
+                Diagnostic(ErrorCode.ERR_MostSpecificImplementationIsNotFound, "I3").WithArguments("I1<string?>.M1()", "I2.I1<string>.M1()", "I3.I1<string>.M1()").WithLocation(2, 19)
+                );
+
+            test1 = compilation3.GetTypeByMetadataName("Test1");
+
+            Assert.Equal(new[] { "I2", "I1<System.String>", "I3", "I1<System.String?>" },
+                         test1.AllInterfacesNoUseSiteDiagnostics.Select(i => i.ToTestDisplayString()));
+
+            Assert.Null(test1.FindImplementationForInterfaceMember(test1.AllInterfacesNoUseSiteDiagnostics[1].GetMember("M1")));
+            Assert.Null(test1.FindImplementationForInterfaceMember(test1.AllInterfacesNoUseSiteDiagnostics[3].GetMember("M1")));
+        }
+
+        [Fact]
+        public void MethodImplementationInDerived_16()
```

Comment:
More generally, it would be good to have a comment or difference in the test name to tell me what the actual difference in the test code is without having to diff the two tests. For 16 and 17, I literally had to go run these through a text differ because I could not spot that the difference between these tests was the location of the nullable directives.